app/components/GameOverlaySettings.tsx

```diff
@@ -61,6 +67,16 @@ export default class GameOverlaySettings extends TsxComponent<{}> {
     });
   }
 
+  get sliderAutohideMetadata() {
+    return metadata.slider({
+      title: $t('Overlay autohide timeout'),
```

Comment:
maybe something like `Fade Time (s)` and `description: $t('Set to 0 to always appear on screen')`